esx_service/vmdk_ops.py

```diff
@@ -708,7 +695,10 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     Returns None (if all OK) or error string
     """
     logging.debug("config_path=%s", config_path)
-    vm_datastore = get_datastore_name(config_path)
+    error_info, vm_datastore = vmdk_utils.get_datastore_name_from_config_path(config_path)
```

Comment:
I see. If that's the case, I agree with you we don't seem to have better choice for this.